src/SelfService.Tests/Infrastructure/Persistence/TestMembershipRepository.cs

```diff
@@ -62,10 +67,64 @@ public async Task cancel_keeps_membership_for_capability_with_single_member()
         var inserted = await dbContext.Memberships.ToListAsync();
         Assert.Contains(member, inserted, new MembershipComparer());
 
-        await repo.Cancel(member.CapabilityId, member.UserId);
+        await repo.CancelWithCapabilityId(member.CapabilityId, member.UserId);
         await dbContext.SaveChangesAsync();
 
         var remaining = await dbContext.Memberships.ToListAsync();
         Assert.Contains(member, remaining, new MembershipComparer());
     }
+
+    [Fact]
+    [Trait("Category", "InMemoryDatabase")]
+    public async Task deactivated_member_cleaner_removes_deactivated_users(){
```

Comment:
Test looks good, but feels like it is placed in the wrong file. I believe this file is for testing the membershiprepository.

Maybe we need to make a test file like TestMembershipCleaner.cs